src/makeInfinityAdder.js

```diff
@@ -3,8 +3,23 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(sum = 0) {
```

Comment:
The default value for the `sum` parameter is set to `0`. According to the task instructions, there is no need to set a default value for the `adder` function argument. Consider removing the default value.